tests/Morphir/Snowpark/FunctionMappingsTests.elm

```diff
@@ -28,9 +28,11 @@ functionMappingsTests =
         customizationOptions = {functionsToInline = Set.empty, functionsToCache = Set.empty}
         (calculatedContext, _ , _) = MappingContext.processDistributionModules testDistributionName testDistributionPackage customizationOptions
         columnsObjects = Dict.fromList [ (FQName.fromString "UTest:MyMod:Emp" ":", "empColumns") ]
-        ctx = { emptyValueMappingContext | typesContextInfo = calculatedContext
-                                         , packagePath = Path.fromString "UTest"
-                                         , dataFrameColumnsObjects = columnsObjects}
+        ctx = 
```

Comment:
Same comment as above with the "update record" syntax

We should not align new assignments to the open brace but to the `|` token